packages/OmegaPrint-Tests.package/TestComment.class/README.md

```diff
@@ -0,0 +1,7 @@
+TestComment is a class which holds some tests concerning comments. It tests from different starting points to see whether comments are removed in between or after statements but also before, after and in between a method.
+The tests are not as small as the other ones as comments are a more complex topic. Comments are not yet fully implemented.
+Also newlines are removed which are a similar problem as comments. 
```

Comment:
which **is** a similar problem (ist ja nur eins) oder which are similar problem**s**

Reply:
Die newlines sind ein ähnliches Problem? was sagen die anderen? an der Stelle beziehe ich mich ja auf mehrere?